build/release/Jenkinsfile

```diff
@@ -17,14 +17,15 @@ pipeline {
       steps {
         script {
           docker.build('maven').inside {
+            def publishingUri = "https://central.sonatype.com/publishing/deployments"
             def targetBranch = "${env.BRANCH_NAME}-${env.BUILD_NUMBER}"
             sh "git config --global user.name 'ivy-team'"
             sh "git config --global user.email 'info@ivyteam.ch'"
             sh "git checkout -b ${targetBranch}"
+            sh "git tag -l | xargs git tag -d"
 
             withCredentials([string(credentialsId: 'gpg.password.axonivy', variable: 'GPG_PWD'), file(credentialsId: 'gpg.keystore.axonivy', variable: 'GPG_FILE')]) {
               sh "gpg --batch --import ${env.GPG_FILE}"
-              def dryRun = isReleaseBranch() ? '' : '-DdryRun=true'
```

Comment:
YAGNI: we have a pre-condition, that only lets /release branches run this pipeline step.